### PR TITLE
Updated Python install instructions

### PR DESCRIPTION
**Pull request by yosefh**
Updated windows method for modifying PATH and added info about latest Python version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,10 +44,11 @@ Contributing
...
-3. Install [VC2008 Express](http://download.microsoft.com/download/A/5/4/A54BADB6-9C3F-478D-8657-93B3FC9FE62D/vcsetup.exe) (alt: mingw)
-4. Install [GitHub for windows](http://windows.github.com/) and get the code
-5. `cd` to the anyway directory
+2. If Python is already installed and its version is lower than 2.7.9 update to a version >= 2.7.9 OR install [setuptools](http://www.lfd.uci.edu/~gohlke/pythonlibs/#setuptools) & [pip](http://www.lfd.uci.edu/~gohlke/pythonlibs/#pip)(package manager) 
+3. Update the PATH to include a reference to the Python and Python/scripts directories. (Replace C:\Python27 with your path to Python) Command Line: Run this command with Administrator privileges: `SETX /M PATH "%PATH%";C:\Python27;C:\Python27\Scripts`. The new path will be available in the next opened terminal. GUI: Control Panel > System Properties > Advanced > Environment Variables > System Variables > Path > Edit > Add ;C:\Python27;C:\Python27\Scripts to the end of the line.
```
Comment (danielhers) [inline README.md]: Surround code and values to be added with backticks (`)

**Comment by danielhers** [comment]
So it works on Windows now? :)

**Comment by omerxx** [comment]
@aviklai is in charge of the windows operation :)
He made it work and several others did too

**Comment by danielhers** [comment]
So now it's clear from the README how to do it?

**Comment by aviklai** [comment]
I think it is. I know that that some people had problems with the path so it is much clearer now.

**Pull request merged**